Classes/Controller/CompanyController.php

```diff
@@ -187,7 +187,7 @@ public function createAction(Company $company): void
     {
         $this->deleteUploadedFilesOnValidationErrors('company');
 
-        /** @var FeUser $feUser */
+        /** @var FrontendUser $feUser */
```

Comment:
Bitte beachte, dass ich hier ein eigenes FeUser-Repo habe und auch ein eigenes FeUser DomainModel. Wenn Du das in FrontendUser umbauen möchtest, dann ändere bitte auch FeUserRepository in FrontendUserRepository

Reply:
Ich habe  Zwecks Kompatibilität die Namen bei $feUser und auch im Model etc. so belassen. Das FeUserRepository ist aber aktualisiert und nutzt ebenfalls den ExtBase FrontendUser